liquidctl/driver/smart_device.py

```diff
@@ -695,7 +694,170 @@ def parse_fan_info(msg):
         return sorted(ret)
 
 
+class RGBController(_BaseSmartDevice):
```

Comment:
The class name is exposed as part of the API, so it would be better to have a minimally more descriptive name. How about `NzxtRgbController`?